tools/docker/Dockerfile

```diff
@@ -41,6 +41,7 @@ FROM build-base
 # Install all Asterinas dependent packages
 RUN apt update && apt-get install -y --no-install-recommends \
     bridge-utils \
+    clang \
```

Comment:
Clang should be installed in the OSDK docker image as it is a dependency of an OSDK feature. So you should add this line in [`osdk/tools/docker/Dockerfile`](https://github.com/asterinas/asterinas/blob/main/osdk/tools/docker/Dockerfile).

Reply:
Updated